### PR TITLE
Configured for maven central

### PR DESCRIPTION
**Pull request by sifraser**
+ set up plugin for the Gradle plugin repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ buildscript {
     }
 }
 
-apply plugin: 'vdm'
+apply plugin: 'com.anaplan.engineering.vdm'
```
Comment (donbex) [inline README.md:89]: Any reason not to use the `plugin` block?

File: README.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ buildscript {
...
```
Comment (donbex) [inline README.md:124]: Same.

File: examples/AlarmSL/build.gradle [hunk 3]
```diff
@@ -10,4 +10,4 @@ buildscript {
 
 version = '1.0.0'
 
-apply plugin: 'vdm'
\ No newline at end of file
+apply plugin: 'com.anaplan.engineering.vdm'
```
Comment (donbex) [inline examples/AlarmSL/build.gradle:13]: I'll stop marking these.

**Review by donbex — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sifraser) [reply README.md:89]: well it's not actually published yet..

File: README.md (hunk 2 above)
Comment (sifraser) [reply README.md:124]: with these ones, I'm not sure how good the gradle test kit is at passing it thru from the build, so I stuck with tried and trusted

File: examples/AlarmSL/build.gradle (hunk 3 above)
Comment (sifraser) [reply examples/AlarmSL/build.gradle:13]: lol

**Pull request merged**